src/convertToObject.js

```diff
@@ -1,18 +1,22 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceArray = sourceString
+    .split(';\n')
+    .map(element => element.split(':'));
+
+  const stylesObject = {};
+
+  sourceArray.forEach(property => {
```

Comment:
The `forEach` method is the same as using a loop. Use reduce instead.
Also, chain all methods. You don't need to have `sourceArray` and `stylesObject` constants at all.